### ISSUE TITLE
`CoilImage(...)` and Recomposition

### ISSUE THREAD

**Issue by Rahkeen** [chrisbanes/accompanist#42]
I have a usecase where I have a grid of images (loaded over network) and when I click on an image I show a new screen with that that image loaded again in a larger container with some extra metadata. I am having issues updating the composable with a new image url, sometimes it will show the old image. 

Here is gif of the issue:
![coil_image_issue_smaller](https://user-images.githubusercontent.com/2228493/87083645-d004cf80-c1e1-11ea-9c6a-9c00fb50421d.gif)

Basically when I click on the first image it loads and shows correctly, but then when I click on other cells in the grid, it will show the image of the cell I previously clicked.

Anything I am doing wrong? It's recomposing and updating the data passed in, but not always showing the new image.

**Comment by Rahkeen** [comment]
Hmmm, might have been an issue on my end. Gonna close for now 😅.

**Comment by chrisbanes** [comment]
I fixed an issue which sounds like this in #35. You could try the `v0.1.7-SNAPSHOT` if you want to try it out ([info](https://github.com/chrisbanes/accompanist#accompanist-snapshots))

**Comment by Rahkeen** [comment]
Ooh, i'll give it a shot, thanks!
